packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_def.hpp

```diff
@@ -115,7 +106,7 @@ namespace FROSch {
         reduceAll(*this->MpiComm_,REDUCE_MAX,int(buildUniqueMap),ptr(&buildUniqueMapMax));
         
         if (buildUniqueMapMax>0) {
-            if (this->MpiComm_->getRank()==0) std::cout << "FROSch::CoarseSpace : WARNING: We re-build a unique map of the AssembledBasisMap_." << std::endl;
+            FROSCH_NOTIFICATION("FROSch::CoarseSpace",this->MpiComm_->getRank()==0,"We re-build a unique map of AssembledBasisMap_.");
```

Comment:
I _really_ like that this is a macro.  (It would be awesome if these notifications were off by default, but users could enable them at run time via an environment variable.)